db/schema.rb

```diff
@@ -10,52 +10,63 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 20_221_108_212_521) do
+ActiveRecord::Schema[7.0].define(version: 2022_11_15_221027) do
   # These are extensions that must be enabled in order to support this database
-  enable_extension 'plpgsql'
+  enable_extension "plpgsql"
 
-  create_table 'comments', force: :cascade do |t|
-    t.text 'text'
-    t.datetime 'created_at', null: false
-    t.datetime 'updated_at', null: false
-    t.bigint 'user_id', null: false
-    t.bigint 'post_id', null: false
-    t.index ['post_id'], name: 'index_comments_on_post_id'
-    t.index ['user_id'], name: 'index_comments_on_user_id'
+  create_table "comments", force: :cascade do |t|
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "user_id", null: false
+    t.bigint "post_id", null: false
+    t.index ["post_id"], name: "index_comments_on_post_id"
+    t.index ["user_id"], name: "index_comments_on_user_id"
   end
 
-  create_table 'likes', force: :cascade do |t|
```

Comment:
There is a migration error which does not allow the user to run the app. I would recommend to please drop clone the project from your repo and try to run it. Kindly make sure that app has no database error and it runs successfully.
![likes](https://user-images.githubusercontent.com/35504149/202282975-53e2a414-7d0c-4dc3-9589-c96793cab504.png)